charts/datafold/charts/dfshell/values.yaml

```diff
@@ -15,17 +15,20 @@ command:
   - "infinity"
 
 serviceAccount:
-  create: true
-  automount: true
+  create: false
```

Comment:
As part of knowledge sharing. Why don't we want/need the service account anymore?